### PR TITLE
Fixes #19541: Remove automatic roles management

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19541

### CODE REVIEW

File: techniques/system/common/1.0/failsafe.st [hunk 1]
```diff
@@ -64,20 +64,13 @@ bundle common g
       # We would like to use date's "--rfc-3339=second" option here, but it is not available on older OSes (RHEL 3/4, AIX 5...)
       "execRun"                  string => execresult("/bin/date -u \"+%Y-%m-%d %T+00:00\"", "noshell");
 
-# definition of the node roles
-  &NODEROLE&
```
Comment (ncharles) [inline techniques/system/common/1.0/failsafe.st:68]: this will be a problem on relay: it's the NODEROLE that says that it's a policy server

**Review by ncharles — CHANGES_REQUESTED** [review]
this ought to be ok, but failsafe has no way to know if the node is a policy server or not

File: techniques/system/common/1.0/failsafe.st (hunk 1 above)
Comment (amousset) [reply techniques/system/common/1.0/failsafe.st:68]: this information does not seem to be actually used in failsafe

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Pull request merged**